test/update-static-lists.js

```diff
@@ -11,6 +11,7 @@ const ENDPOINTS = {
     "https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest?limit=5000&start=1&sort=market_cap_strict&market_cap_min=10000000",
   COINMARKETCAP_COIN_INFO:
     "https://pro-api.coinmarketcap.com/v2/cryptocurrency/info?id=",
+  SNAPS_REGISTRY_LIST: "https://raw.githubusercontent.com/MetaMask/snaps-registry/main/src/registry.json"
```

Comment:
```suggestion
  SNAPS_REGISTRY_LIST: "https://acl.execution.metamask.io/latest/registry.json"
```
AFAIK we should refrain from downloading directly from `raw.githubusercontent.com`. We can use the hosted version instead!